docker-compose.yml

```diff
@@ -170,17 +173,19 @@ services:
       STORE_PGDATABASE: dev_database
 
       ADMIN_API_PORT: 3001
-      STORE_API: http://store-api:3000
+      STORE_API: http://localhost:3005
```

Comment:
I'm not sure about this change. It will mean the admin api connects to the store api via the users' host machine network. And this at least work wont in MacOS I think, where docker runs containers entirely virtualized without access to the host machine's network.

Same comment about other occurrences of "localhost" in the code changes of this file.

Reply:
Fixed + small patch for admin-frontend.

Inside codes, the url of `store-api` was defined as `$REACT_APP_STORE_BASE_URL/api/`. In the patch, it has another optional environment `$REACT_APP_STORE_API_URL`. When this variable exists, then api url will use this variable.

Old code:
```
process.env.REACT_APP_STORE_BASE_URL + '/api'
```
New code:
```
(process.env.REACT_APP_STORE_API_URL??
        process.env.REACT_APP_STORE_BASE_URL + '/api')
```